cla_public/config/common.py

```diff
@@ -99,7 +99,7 @@ def config_path(x):
 TIMEZONE = "Europe/London"
 
 LAALAA_API_HOST = os.environ.get("LAALAA_API_HOST", "https://prod.laalaa.dsd.io")
-
+# Remove the SMTP bits after the code has been removed
```

Comment:
TODO?

Reply:
Discussed reasons why this is left in the code with Pat (Awaiting Sendgrid removal)